package.json

```diff
@@ -52,19 +51,19 @@
     "validator": "13.7.0"
   },
   "scripts": {
-    "pre-commit": "yarn prettier:check && yarn lint",
+    "pre-commit": "npx tsc --noEmit && yarn prettier:check && yarn lint",
```

Comment:
I think this should go in a `.husky/pre-commit` file with the following content:

```sh
#!/usr/bin/env sh
. "$(dirname -- "$0")/_/husky.sh"

npx tsc --noEmit && yarn prettier:check && yarn lint
```